iphone/Maps/Classes/MapDownloader/MWMBaseMapDownloaderViewController.mm

```diff
@@ -501,6 +501,11 @@ - (IBAction)longPress:(UILongPressGestureRecognizer *)sender
 
 - (void)actionSheet:(UIActionSheet *)actionSheet clickedButtonAtIndex:(NSInteger)buttonIndex
 {
+  if (buttonIndex >= actionSheet.numberOfButtons)
```

Comment:
Ты же ревьюил код и мержил.

На сколько был критичен этот креш? Лучше было бы его оставить и залогать, возможно, там убиваются объекты или что-нибудь более стремное происходит.

> On Apr 8, 2016, at 15:59, Vlad Mihaylenko <notifications@github.com> wrote:
> 
> In iphone/Maps/Classes/MapDownloader/MWMBaseMapDownloaderViewController.mm <https://github.com/mapsme/omim/pull/2779#discussion_r59020627>:
> 
> > @@ -501,6 +501,11 @@ - (IBAction)longPress:(UILongPressGestureRecognizer *)sender
> >  
> >  - (void)actionSheet:(UIActionSheet *)actionSheet clickedButtonAtIndex:(NSInteger)buttonIndex
> >  {
> > +  if (buttonIndex >= actionSheet.numberOfButtons)
> провобовали, не воспроизводится. автор пул реквеста не я.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2779/files/362adb4fcf0abf0f56f27377d9292e1ce953f00f#r59020627>